cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/FieldLineageTable.java

```diff
@@ -481,4 +484,41 @@ private List<Field<?>> getOperationsKey(long checksum) {
     fields.add(Fields.longField(StoreDefinition.FieldLineageStore.CHECKSUM_FIELD, checksum));
     return fields;
   }
+
+  public List<EndPoint> getEndpoints(String namespaceId, long startMillis) throws IOException {
```

Comment:
multiple runs can have the save start time, this method should be changed to take the runid. Looking at StoreDefinition.FieldLineageStore, it also looks like we need to add an index on the runid column to make sure this is performant.

Reply:
Offline discussion: Because the column is a program run id and not the uuid, will filter the right row on runId from the list obtained